src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -460,7 +461,7 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
 
             ProjectRootManager.getInstance(project).contentRoots.forEach {
                 if (it.isDirectory) {
-                    val matchedFile = it.findFileByRelativePath(filePath.toString())
+                    val matchedFile = findByRelativePath(it, filePath)
```

Comment:
Sorry to annoy you again, but I think we could also use `ProjectUtils.tryFindFile` here